### PR TITLE
Add docs to `@test` package

### PR DESCRIPTION
**Pull request by fantix**
This PR also adds the `.git-blame-ignore-revs` file to exlucde changes from #162 in `git blame`, see also: https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

[Sample blame](https://github.com/fantix/core/blame/9c3e42f809b8c63c1c9dd83570c5651d193bfb2b/queue/queue.mbt#L53C3-L59)

- [ ] Depends on #162 
- [ ] Depends on #163

### CODE REVIEW

File: test/assertion.mbt [hunk 1]
```diff
@@ -18,6 +18,20 @@ fn debug_string[T : Debug](t : T) -> String {
...
+///
+/// Returns Ok if the two arguments are equal by value, using `==`; raises an
+/// Error otherwise.
+///
+/// # Examples
+///
+/// ```moonbit
+/// let a = "4" + "2"
+/// let b = "4" + "2"
+/// @test.eq(a, a)?  // this is okay
```
Comment (fantix) [inline test/assertion.mbt:31]: Using `@` in docstrings produces a compiler warning:

```
core/test/assertion.mbt:35:1-47:2 Warning 020: Invalid pragma, unexpected id or invalid properties
```

Is it a bug or feature?

Comment (peter-jerry-ye) [reply test/assertion.mbt:31]: cc @Yoorkin

Comment (Yoorkin) [reply test/assertion.mbt:31]: > Using `@` in docstrings produces a compiler warning:
> 
> ```
> core/test/assertion.mbt:35:1-47:2 Warning 020: Invalid pragma, unexpected id or invalid properties
> ```
> 
> Is it a bug or feature?

It's a bug. The `@test.` should be ignored by the parser, but it's being recognized as pragmas. The pragmas is a new feature that we are planning to introduce today.

**Comment by fantix** [comment]
Closing as being less favored

**Pull request closed without merging**